# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: toolkit/library/rust/shared/Cargo.toml
# Commit: 073990df3224
# Full Hash: 073990df32240a0cda9381678ed133b014741644
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2024-06-17 21:44:16
# Description:
#   Bug 1902801 - Update cubeb-coreaudio-rs to f8a4ec05e5. r=cubeb-reviewers,padenot
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D213930
# ==============================================================================

diff -r f183a6936e48 -r 073990df3224 toolkit/library/rust/shared/Cargo.toml
--- a/toolkit/library/rust/shared/Cargo.toml	Mon Jun 17 12:32:00 2024 +0000
+++ b/toolkit/library/rust/shared/Cargo.toml	Mon Jun 17 12:50:03 2024 +0000
@@ -22,7 +22,7 @@
 profiler_helper = { path = "../../../../tools/profiler/rust-helper", optional = true }
 mozurl = { path = "../../../../netwerk/base/mozurl" }
 webrender_bindings = { path = "../../../../gfx/webrender_bindings" }
-cubeb-coreaudio = { git = "https://github.com/mozilla/cubeb-coreaudio-rs", rev = "1796ace5bdd08ec8baa56bbf7170a08d760c984b", optional = true }
+cubeb-coreaudio = { git = "https://github.com/mozilla/cubeb-coreaudio-rs", rev = "f8a4ec05e5d6200b193eff0b7a5b98e462008c5d", optional = true }
 cubeb-pulse = { git = "https://github.com/mozilla/cubeb-pulse-rs", rev="8678dcab1c287de79c4c184ccc2e065bc62b70e2", optional = true, features=["pulse-dlopen"] }
 cubeb-sys = { version = "0.13", optional = true, features=["gecko-in-tree"] }
 audioipc2-client = { git = "https://github.com/mozilla/audioipc", rev = "3495905752a4263827f5d43737f9ca3ed0243ce0", optional = true }
